README.md

```diff
@@ -1,97 +1,50 @@
-# Quick Start
+# Overview
 
-## Prerequisite
-
-In the latest release, Ape requires:
-
-- Linux or macOS
-- Python 3.8 or later
-
-**Windows**:
-
-1.  Install Windows Subsystem Linux
-    [(WSL)](https://docs.microsoft.com/en-us/windows/wsl/install)
-2.  Choose Ubuntu 20.04 OR Any other Linux Distribution with Python
-    3.8 or later
-
-Check your python command by entering:
-
-```bash
-python3 --version
-```
-
-## Installation
-
-There are 3 ways to install ape: `pipx`, `pip`, or `Docker`.
-
-### via `pipx`
-
-Install `pipx` via their [installation instructions](https://pypa.github.io/pipx/)
-
-Then install `ape` via
+**Ape Framework** is an easy-to-use, beginner friendly Web3 development tool. 
+Users can compile, test, and interact with smart contracts all in one command line session.
+With our **modular plugin system**, Ape supports multiple contract languages and chains.
 
-```bash
-pipx install eth-ape
-```
-
-To install Ape and a list of common, recommended plugins at the same time, do:
+Ape is built by [ApeWorX LTD](https://www.apeworx.io/).
 
-```bash
-pipx install eth-ape'[recommended-plugins]'
-```
+Join our [ApeWorX Discord server](https://discord.gg/apeworx) to stay up to date on new release, plugins and tutorials.
 
-### via `pip`
+If you want to just get started, jump down to our Quick Start guide!
 
-**Suggestion**: Create a virtual environment using `virtualenv` or `venv.`
+# Documentation
```

Comment:
It is problematic to have multiple top-level headers actually...
There should only be 1 header with a single `#` in the entire document.